src/psyclone/psyir/transformations/loop_swap_trans.py

```diff
@@ -38,8 +38,10 @@
 
 ''' This module provides the loop swap transformation.'''
 
-from psyclone.psyir.nodes import Call, CodeBlock
-from psyclone.psyir.transformations import LoopTrans, TransformationError
+from psyclone.psyir.nodes import Call, CodeBlock, Reference
+from psyclone.psyir.transformations.loop_trans import LoopTrans
+from psyclone.psyir.transformations.transformation_error import \
+        TransformationError
 
 
 class LoopSwapTrans(LoopTrans):
```

Comment:
Not strictly this PR but pylint complains that the `node` argument to `validate` has been renamed `node_outer` in this class. Similarly it complains about the `apply` method too.

Reply:
Renamed both to "node" to clean the pylint errors.